Makefile

```diff
@@ -322,11 +321,16 @@ telemetrygen:
 
 .PHONY: update-otel
 update-otel:$(MULTIMOD)
-	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m stable --commit-hash $(OTEL_STABLE_VERSION)
-	git add . && git commit -s -m "[chore] multimod update stable modules"
-	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m beta --commit-hash $(OTEL_VERSION)
-	git add . && git commit -s -m "[chore] multimod update beta modules"
+	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector-contrib -m stable 
+	git add . && git commit -s -m "[chore] multimod update stable modules" ; \
+	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector-contrib -m beta 
+	git add . && git commit -s -m "[chore] multimod update beta modules" ; \
+	sed -i -E -e "s/v[0-9]+\.[0-9]+\.[0-9].*$\/v$(COREVERSION)/" -e "5,6s/[0-9]+\.[0-9]+\.[0-9]/$(COREVERSION)/" ./cmd/otelcontribcol/builder-config.yaml 
```

Comment:
It might be useful to have some kind of explanation for the regexs here

Reply:
Yes, of course.
Actually we're scanning for any version-indices like e.g. `v.0.97.0` with the expression: `v[0-9]+\.[0-9]+\.[0-9].*$`
And replace this expression + everything to the end of the line, because version-indices are placed there and have to be extended because of versions like `0.97.0-xyz-49271j2n3`.

Then it gets replaced by the grepped version via `v$(COREVERSION)`.

The next expression is only for the title lines in which the version is provided without the prefix "v". So we can just replace the version-indices in this lines.